traffic_monitor/datareq/cachestate.go

```diff
@@ -170,34 +173,37 @@ func createCacheStatuses(
 			ConnectionCount:        connections,
 			Status:                 &status,
 			StatusPoller:           &statusPoller,
+			IPV4Available:          &ipv4,
+			IPV6Available:          &ipv6,
+			CombinedAvailable:      &combinedStatus,
 		}
 	}
 	return statii
 }
 
-func cacheStatusAndPoller(server tc.CacheName, serverInfo tc.TrafficServer, localCacheStatus cache.AvailableStatuses) (string, string) {
+func cacheStatusAndPoller(server tc.CacheName, serverInfo tc.TrafficServer, localCacheStatus cache.AvailableStatuses) (string, string, bool, bool, bool) {
```

Comment:
A function-level comment documenting what the return values are would be helpful